### ISSUE TITLE
Autoload script not removed

### ISSUE THREAD

**Issue by IceflowRE** [imjp94/gd-plug#28]
> autoload script references in project setting are not cleared as plugin uninstalled.
    Workaround: Manually remove autoload scripts from project setting

Can elaborate what does this mean?
So if a plugin has an autoload, that autoload is not removed on uninstalling the plugin?

**Comment by imjp94** [comment]
Yes, but it is just the "reference" of autoload that got left in `project.godot`, while the script itself will be properly removed.

For example, this is what might happen to `project.godot` after uninstalling a plugin with autoloads:
```cfg
[autoload]

Yafsm="*res://addons/imjp94.yafsm/YAFSM.gd" # Doesn't exist anymore
```
And editor will start popping warnings about missing autoload script

**Comment by IceflowRE** [comment]
OH, just tried gd-plug and plugins are not enabled after installation or disabled before uninstall? Is there any reason why you do not enable/disable it?
This would avoid the autoload problem at all.

**Comment by imjp94** [comment]
Hmm... Can't quite remember why I didn't disable the autoloads programmatically before uninstalling, maybe it is hard to locate autoloads owned by plugin that about to be uninstalled(I guess?)
Might look into this again if I have time

**Comment by IceflowRE** [comment]
> Hmm... Can't quite remember why I didn't disable the autoloads programmatically before uninstalling, maybe it is hard to locate autoloads owned by plugin that about to be uninstalled(I guess?) Might look into this again if I have time

You do not have to remove the autoloads of plugins, a plugin should do that on enabling/disabling themself.

**Comment by imjp94** [comment]
I see, thanks for the info, will give it a try =)
